toplevel/workerLoop.mli

```diff
@@ -8,7 +8,7 @@
 (*         *     (see LICENSE file for the text of the license)         *)
 (************************************************************************)
 
-(* Register a STM worker *)
+(* Register a STM worker of a given executable name *)
 val start :
   init:(unit -> unit) ->
-  loop:(unit -> unit) -> unit
+  loop:(unit -> unit) -> string -> unit
```

Comment:
I don't like this string argument, since it may get out of sync with `T.name` (see AsyncTaskQueue.Make argument).
IMO, you should get the name out of `Sys.executable_name` or something like that.

Reply:
I made this by consistency with `coqtop` and `coqc` whose name for `--help` is similarly hard-wired in the usage text. We could indeed alternatively use `Sys.executable_name` in each of the cases. (For instance, I tested with `ocamlc` and it uses `Sys.executable_name`. I tested with `gcc` and it uses a hard-wired name, from what I conclude that there is somehow an arbitrary choice.)

If no further comments about that in the discussion, I shall use `Sys.executable_name`.